helpers/signatureVerification.ts

```diff
@@ -0,0 +1,121 @@
+/* 
+Helper to check if signature is a single signature
+*/

```

Comment:
Try to follow the existing documentation style using 
```
/**
 * description
 */
```
```suggestion
/**
 * Check if the format of input matches that of a single signature contract
 */
```